### PR TITLE
Add PyCon Russia 2017

### PR DESCRIPTION
**Pull request by redapple**
Fixes https://github.com/pyvideo/data/issues/328

Data from https://github.com/pyconru/pycon.ru/tree/carcade/pages/program/content, https://github.com/pyconru/pycon.ru/blob/carcade/pages/program/schedule/data.ru.yaml and https://www.youtube.com/playlist?list=PLRdS-n5seLRrcQgUdFr0OG_XP-ZU7FcPS

### CODE REVIEW

File: pycon-ru-2017/videos/about-the-analyst-and-silver-bullets.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "In my talk I will touch upon Rambler/top-100 relaunch, available instruments for realime analytics' system developement and about our experience of exchanging batch-calculations for realtime pipelines. I'll introduce you to the architecture we used for both data processing approaches and will pay attention to all the components. Then we'll duscuss peculiar properties of python that matter when implementing data procession algirithms for HIVE, touch upon fundamental problems of storing aggregates as well as pros and cons of alternative approach. I'll bring you details on how we use Spark for changing sessions calculation, we'll discuss emerging issues and existing solutions. In the end i'll show you our results of performance tests and will introduce you to several pitfalls we've met on our way.\n",
```
Comment (Daniel-at-github) [inline pycon-ru-2017/videos/about-the-analyst-and-silver-bullets.json]: Typo: algirithms => algoritms

File: pycon-ru-2017/videos/another-yandex-search.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "Яндекс — поисковая компания. Но хорошо находить информацию нужно не только в Интернете. У нас внутри есть огромный интранет – большое количество внутренних сервисов, по которым тоже нужно эффективно искать. Мы любим Python и активно его используем для того, чтобы сделать поиск удобным, быстрым и максимально точно отвечающим на вопросы сотрудников. У нас довольно интересная инфраструктура – есть компонентны и на Django, и на Tornado. Мы активно используем Celery с разными брокерами. Взаимодействуем с внутренними инструментами и API, чтобы переиспользовать наработки большого Поиска. Этот же механизм стал недавно использоваться для поиска по данным организаций внутри сервисов Яндекс.Коннекта. Я хочу рассказать, как все эти компоненты взаимодействуют и какие есть интересные особенности.\n",
+  "duration": 1781,
+  "recorded": "2017-07-17",
```
Comment (Daniel-at-github) [inline pycon-ru-2017/videos/another-yandex-search.json]: Add language
~~~ json
  "language": "rus",
~~~

File: pycon-ru-2017/videos/lightning-talks-na-pycon-russia-2017.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "Lightning Talks на PYCON RUSSIA 2017",
+  "duration": 2819,
```
Comment (Daniel-at-github) [inline pycon-ru-2017/videos/lightning-talks-na-pycon-russia-2017.json]: Add language
~~~ json
  "language": "rus",
~~~

File: pycon-ru-2017/videos/load-testing-with-a-touch-of-python.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "This talk can be useful for all the listeners who face the problem of measuring application performance and capacity and decide which is the best tool for their needs. It's great when a simple tool like ab or wrk is enough to ensure your application performance. And what if you need a sophisticated scenario to test your service? What should you choose? JMeter because of hundred of plugins? Tsung because of easy scaling? Gatling because of support for different protocols? And why not use Python for load test scenario? I'll share the experience of load testing with Yandex Tank and BFG - load generator which supports user-defined scripts in Python. We'll focus on one recovery test realized with BFG. I'll show how Python scenarios can be used for load, performance and another non-functional testing. Why it helped to make load testing a part of a development cycle. And how BFG became more popular than jMeter in our daily practice.\n",
+  "duration": 2060,
```
Comment (Daniel-at-github) [inline pycon-ru-2017/videos/load-testing-with-a-touch-of-python.json]: Add language
~~~ json
  "language": "rus",
~~~

File: pycon-ru-2017/videos/why-is-python-slow.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "In this talk, I will cover one of the usual topics of discussions of deficiencies of the Python language, notably it's speed.  \nWe've spent last decade improving on the performance of the language (but not the default implementation). What we've learned so far is not the common interpretation of that question - things like dynamic dispatch, dynamic typing and interpreter can all be worked around.  \nWhat we've learned are the places in the Python language which make unnecessary string copies, but are easier to write, a lot of \"pythonic\" constructs that cannot be implemented efficiently and a lot of quirks that make some constructs slow for no good reason. Additionally we're overwhelmed by the sheer size of the \"simple\" language and the necessity of supporting C extensions.  \nThis talk will cover detailed view of those problems and some potential remedies.\n",
+  "duration": 1889,
```
Comment (Daniel-at-github) [inline pycon-ru-2017/videos/why-is-python-slow.json]: Add language
~~~ json
  "language": "eng",
~~~

File: pycon-ru-2017/videos/elegant-solutions-for-everyday-python-problems.json [hunk 6]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "Are you an intermediate Python programmer who becomes frustrated by copy and pasting or duplicating code? Luckily, the Python programming language provides us with a unique set of tools to make our code more elegant, readable, and reduce duplication. In this talk, I’ll share common anti-patterns in python programs and show you practical pythonic solutions for supercharging your code with tools like Decorators, Context Managers, Mixins, and Lambdas. \n",
+  "duration": 2620,
```
Comment (Daniel-at-github) [inline pycon-ru-2017/videos/elegant-solutions-for-everyday-python-problems.json]: Add language
~~~ json
  "language": "eng",
~~~

File: pycon-ru-2017/videos/gradual-typing-of-production-applications.json [hunk 7]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "Thanks to Python's dynamic nature and minimal boilerplate, it's easy to start a project. But how do you keep it maintainable and easy to understand as it grows beyond a million lines of code? Come learn how annotating it with types makes your program better for human readers and more robust during execution. No prior Haskell experience required!\n\nIn an hour I'm going to demonstrate real world problems of big software projects that type annotations solve. The talk is going to explain how you can introduce typing to an already mature application, what is a sensible workflow to follow and how to solve common gotchas.  No ducks were harmed in the making of this talk.\n",
+  "duration": 3686,
```
Comment (Daniel-at-github) [inline pycon-ru-2017/videos/gradual-typing-of-production-applications.json]: Add language
~~~ json
  "language": "eng",
~~~

**Comment by Daniel-at-github** [comment]
Add russian language to all ...

~~~ json
  "language": "rus",
~~~

... except three:

pycon-ru-2017/videos/why-is-python-slow.json
pycon-ru-2017/videos/elegant-solutions-for-everyday-python-problems.json
pycon-ru-2017/videos/gradual-typing-of-production-applications.json

~~~ json
  "language": "rus",
~~~

**Comment by redapple** [comment]
Thanks for the review @Daniel-at-github . Amended.

**Pull request merged**